jinx.el

```diff
@@ -718,14 +720,14 @@ If CHECK is non-nil, always check first."
        (funcall fun 'save key (if (> len 1) (substring choice 1) word))
        (jinx--recheck-overlays))
       ((guard (not (equal choice word)))
-       (jinx--correct-replace overlay choice)))))
+       (jinx--correct-replace start end choice)
+       (when post-replace
+         (funcall post-replace))))))
```

Comment:
Instead of the post-replace function I would prefer if `jinx--correct-replace` searches for Jinx overlays between start and end and simply deletes them.